src/vec8bit.c

```diff
@@ -5733,7 +5732,7 @@ Obj FuncTRANSPOSED_MAT8BIT( Obj self, Obj mat)
                         val = vals[0];
 
                     /* set entry */
-                    ptr = BYTES_VEC8BIT(ELM_GF2MAT(tra, nstart + j)) + imod;
+                    ptr = BYTES_VEC8BIT(ELM_MAT8BIT(tra, nstart + j)) + imod;
```

Comment:
But in line 4036, a GF2MAT is accessed... Of course internally the two functions do the same, but still.